--- a/arrays/@stk_dataframe/stk_dataframe.m
+++ b/arrays/@stk_dataframe/stk_dataframe.m
@@ -175,11 +175,6 @@
 
 x = class (x, 'stk_dataframe');
 
-try  %#ok<TRYNC>
-    % Starting with Matlab R2014b, graphics handles are objects
-    superiorto ('matlab.graphics.axis.Axes');
-end
-
 end % function
 
 
--- a/arrays/@stk_factorialdesign/stk_factorialdesign.m
+++ b/arrays/@stk_factorialdesign/stk_factorialdesign.m
@@ -103,12 +103,7 @@
     % "factorial design" object
     x = struct ('levels', {levels});
     x = class (x, 'stk_factorialdesign', df);
-    
-    try
-        % Starting with Matlab R2014b, graphics handles are objects
-        superiorto ('matlab.graphics.axis.Axes');
-    end
-    
+        
 end % if
 
 end % function
