--- a/arrays/@stk_dataframe/stk_dataframe.m
+++ b/arrays/@stk_dataframe/stk_dataframe.m
@@ -199,11 +199,6 @@
 
 x = class (x, 'stk_dataframe');
 
-try  %#ok<TRYNC>
-    % Starting with Matlab R2014b, graphics handles are objects
-    superiorto ('matlab.graphics.axis.Axes');
-end
-
 end % function
 
 
--- a/arrays/@stk_factorialdesign/stk_factorialdesign.m
+++ b/arrays/@stk_factorialdesign/stk_factorialdesign.m
@@ -112,11 +112,6 @@
     x = struct ();  x.levels = levels;
     x = class (x, 'stk_factorialdesign', df);
     
-    try %#ok<TRYNC>
-        % Starting with Matlab R2014b, graphics handles are objects
-        superiorto ('matlab.graphics.axis.Axes');
-    end
-    
 end % if
 
 end % function
