--- a/stk_init.m
+++ b/stk_init.m
@@ -53,9 +53,6 @@ switch command
     case 'pkg_unload'
         stk_init__pkg_unload (root);
         
-    case 'prune_mole'
-        stk_init__config_mole (root, false, true);  % prune, but do not add to path
-        
     case 'clear_persistents'
         % Note: this implies munlock
         stk_init__clear_persistents ();
@@ -187,9 +184,6 @@ end
 % Add STK folders to the path
 addpath (path{:});
 
-% Selectively add MOLE subdirectories to compensate for missing functions
-stk_init__config_mole (root, true, false);  % (add to path, but do not prune)
-
 end % function
 
 
@@ -285,78 +279,3 @@ s = strrep (s, '+', '\+');
 s = strrep (s, '.', '\.');
 
 end % function
-
-
-function stk_init__config_mole (root, do_addpath, prune_unused)
-
-mole_dir = fullfile (root, 'misc', 'mole');
-isoctave = (exist ('OCTAVE_VERSION', 'builtin') == 5);
-
-if isoctave
-    recursive_rmdir_state = confirm_recursive_rmdir (0);
-end
-
-opts = {root, mole_dir, do_addpath, prune_unused};
-
-% Provide missing octave functions for Matlab users
-% TODO: extract functions that are REALLY needed in separate directories
-%       and get rid of the others !
-if (exist ('OCTAVE_VERSION', 'builtin') ~= 5)  % if Matlab
-    if do_addpath
-        addpath (fullfile (mole_dir, 'matlab'));
-    end
-elseif prune_unused
-    rmdir (fullfile (mole_dir, 'matlab'), 's');
-end
-
-% graphics_toolkit
-%  * For Octave users: available in Octave >= 4.0.0
-%  * For Matlab users: there is no function named graphics_toolkit in Matlab.
-%    Our implementation returns either 'matlab-jvm' or 'matlab-nojvm'.
-install_mole_function ('graphics_toolkit', opts{:});
-
-% quantile
-%  * For Octave users: available in Octave >= 4.0.0
-%  * For Matlab users: quantile is missing from Matlab itself, but it provided
-%    by the Statistics toolbox if you're rich enough to afford it.
-install_mole_function ('quantile', opts{:});
-
-% cleanup
-if isoctave
-    confirm_recursive_rmdir (recursive_rmdir_state);
-end
-
-end % function
-
-
-function install_mole_function (funct_name, ...
-    root, mole_dir, do_addpath, prune_unused)
-
-function_dir = fullfile (mole_dir, funct_name);
-
-w = which (funct_name);
-
-if (isempty (w)) || (~ isempty (strfind (w, root)))  %#ok<STREMP> % if the function is absent
-    
-    function_mfile = fullfile (function_dir, [funct_name '.m']);
-    
-    if exist (function_dir, 'dir') && exist (function_mfile, 'file')
-        
-        % fprintf ('[MOLE]  Providing function %s\n', function_name);
-        if do_addpath
-            addpath (function_dir);
-        end
-        
-    else
-        
-        warning (sprintf ('[MOLE]  Missing function: %s\n', funct_name));
-        
-    end
-    
-elseif prune_unused && (exist (function_dir, 'dir'))
-    
-    rmdir (function_dir, 's');
-    
-end
-
-end % function
